.github/workflows/loristest.yml

```diff
@@ -0,0 +1,92 @@
+name: LORIS Test Suite
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        matrix:
+            testsuite: ['integration']
+            php: ['8.1','8.2', '8.3']
+            ci_node_index: [0]
+
+            include:
+            # add a variable but do not display it in the job's name
+            - ci_node_total: 1
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php }}
+        extensions: zip, php-ast
+
+
+    - name: Clone Loris repo
+      run: git clone https://github.com/aces/Loris.git
+
+    - name: Override Test Files
+      run: |
+        cp test/Dockerfile.test.php8 Loris/
+        cp test/docker-compose.yml Loris/
+        cp test/phpunit.xml Loris/test/
+        cp test/RB_SQL/* Loris/raisinbread/RB_files/
```

Comment:
Could we select here the RB SQL tables we want to source? Technically, we only need the following tables:

- Config
- ConfigSettings
- ImagingFileTypes
- Project
- Visit_Windows
- all the `bids-*` tables
- candidate
- cohort
- mri_protocol_* tables
- mri_scan_type
- parameter_type (however, this could be populated automatically when running the pipeline)
- permissions* (I would guess this would be needed when using the API)
- project_cohort_rel
- psc
- session
- user_perm_rel
- user_project_rel
- user_psc_rel
- users
- visit
- visit_project_cohort_rel


Purposely skipping the following tables:
- feedback_mri_* (don't need feedbacks for LORIS-MRI)
- files (would use a very lightweight version of this table)
- files_intermediary
- files_qcstatus (QC so not really used by LORIS-MRI)
- electrophysiology_uploader (not sure if this is needed)
- hed_schema (not sure if this has data or has set up information, would need @jeffersoncasimir to decide)
- hed_schema_nodes (same, @jeffersoncasimir)
- hrrt_* (empty tables in RB)
- mri_parameter_form ( I don't think there is any use of that table for LORIS-MRI)
- mri_processing_protocol (empty table in RB)
- mri_scanner (has scanner data but the pipeline is supposed to create that if it does not exist so maybe we can ensure the logic of creating the scanner works if we skip this table?)
- mri_upload* (could have a lightweight version of it or keep it empty)
- mri_violations_log (could have a lightweight version of it or keep it empty)
- notification_spool (we don't really need all of that data)
- parameter_files (could have a lightweight version of it or keep it empty)
- physiological* tables (could have a lightweight version of it or keep it empty and the script will populate those)
- server_processes (could probably keep that empty)
- tarchive* (would have a lightweight version of it with a couple of files)
- violations_resolved (could have a lightweight version of it or keep it empty)

Reply:
yes, you can do it override any files under the RB_SQL folder.